### DISCUSSION TITLE
const immutable values

### DISCUSSION THREAD

**Discussion by simongoldstone** [dotnet/csharplang#6469] [Language Ideas] [answered]
Introduction of immutable values within method block using const keyword, similar to that of typescript, for example.

int MyMethod()
{
    const int myVal = 123;   // explicit type def
    const myVal2 = "Str123";  // implicit type, like var assignment

}

**Comment by simongoldstone** [comment]
Perhaps could also be the let keyword

**Comment by FaustVX** [comment, marked as answer]
There is already readonly local proposal : #5057

**Reply by simongoldstone** [reply]
Ahh, I missed that. Perfect, thanks!
